plugins/SitesManager/stylesheets/SitesManager.less

```diff
@@ -139,6 +146,87 @@ td.editable-site-field:hover {
             width: unset !important;
             float: unset !important;
         }
+
+        a:not(.active) {
+            text-decoration: none !important;
+            color: @color-black !important;
+
+            &:hover {
+                background-color: @color-gray-bright !important;
+                opacity: unset;
+            }
+
+            &:focus {
+                background-color: @color-jetstream !important;
+                opacity: unset;
+            }
+        }
+
+        a.active {
+            color: @color-green-piwik !important;
+            text-decoration: none !important;
+            font-weight: bold;
+
+            &:hover {
+                opacity: unset;
+            }
+        }
+
+        .indicator {
+            border-bottom: 2px solid @color-green-piwik;
+            z-index: 10;
+            bottom: 0.3rem;
+        }
+
+        .tab {
+            text-transform: unset;
+
+            a {
+                font-size: 16px;
+            }
+        }
+    }
+
+    .no-data-screen-ul-tabs::after {
+        border-bottom: 0.5px solid @color-gray-400;
+        width: 100%;
+        content: '';
+        display: block;
+        overflow: hidden;
+        bottom: 0.3rem;
+        z-index: 2;
+        position: absolute;
+    }
+
+    .no-data-footer {
+        position: sticky;
+        width: inherit;
+        bottom: 0;
+        background: @color-white;
+
+        .col {
+            width: unset !important;
+        }
+
+        @media only screen {
+            @media (max-width: 800px) {
+                .m-bottom-1 {
+                    margin-bottom: 1rem;
+                }
+            }
+        }
+    }
+
+    #start-tracking-data-header {
+        font-weight: bold;
+    }
+
+    .underline a {
+        text-decoration: underline;
+    }
```

Comment:
@AltamashShaikh @Stan-vw @Javi-Ormaechea Is it really on purpose that the links in the introduction text are always underlined? In general there is no problem with underlining links, but we do not have that anywhere in Matomo. And it imho is a "break" in the layout to start underlining links only in a few random places.
There are even links on the same page that aren't underlined. If we want to start having underlined links we should consider a global decision and underline them everywhere - or nowhere!

Reply:
@Stan-vw Its due to design
![Screenshot from 2023-05-10 12-31-50](https://github.com/matomo-org/matomo/assets/14902277/3d890c5d-c5aa-4d65-961c-118cb7e525b0)

Reply:
I've seen that. I just wanted to point out this inconsistency, which we imho should try to avoid.

Reply:
@sgiehl Removed underline for now, we can later implement it globally :+1:

Reply:
Thanks for pointing out the inconsistency. As Altamash said, let's adjust global hyperlink font in a separate ticket rather than creating inconsistencies 👍

Reply:
Thanks Stefan, I agree with all these comments. We should definitely implement this globally.